GenderPayGap.WebUI/Views/Register/OrganisationType.cshtml

```diff
@@ -31,24 +31,57 @@
                         @Html.ValidationMessageFor(model => model.SectorType, null, new {@class = "error-danger"})
                     </legend>
 
+                    <div class="multiple-choice">
+                        <input id="public" type="radio" name="SectorType" value="@(SectorTypes.Public)">
+                        <label for="public">
+                            Public sector organisation
+                        </label>
+                    </div>
                     <div class="multiple-choice">
                         <input id="private" type="radio" name="SectorType" value="@(SectorTypes.Private)">
                         <label for="private">
-                            Private limited company,<br/>
-                            Limited liability partnership,<br/>
-                            Charity,<br/>
+                            Private limited company
+                        </label>
+                    </div>
+                    <div class="multiple-choice">
+                        <input id="publicLimitedCompany" type="radio" name="SectorType" value="@(SectorTypes.Private)">
+                        <label for="publicLimitedCompany">
                             Public limited company
                         </label>
                     </div>
                     <div class="multiple-choice">
-                        <input id="public" type="radio" name="SectorType" value="@(SectorTypes.Public)">
-                        <label for="public">
-                            Public sector organisation
+                        <input id="limitedLiabilityPartnership" type="radio" name="SectorType" value="@(SectorTypes.Private)">
+                        <label for="limitedLiabilityPartnership">
+                            Limited liability partnership
                         </label>
                     </div>
+                    <div class="multiple-choice">
+                        <input id="charity" type="radio" name="SectorType" value="@(SectorTypes.Private)">
+                        <label for="charity">
+                            Charity
+                        </label>
+                    </div>
+
                 </fieldset>
             </div>
-            <br/>
+            @(Html.GovUkInsetText(new InsetTextViewModel
+            {
+                Html = @<div class="govuk-!-font-size-24">
+                           <p>
+                               If you are not sure what type of organisation you are please read our
+                               <a class="govuk-link" href="https://www.gov.uk/guidance/gender-pay-gap-reporting-overview#relevant-employer">
+                                   guidance</a>.

```

Comment:
If the closing `</a>` tag is on the next line then the link text has an extra unwanted space at the end. Any ideas why? Or how to get round this?

Reply:
It seems to only do that if you have something after the `</a>` tag (and it doesn't matter whether the something is preceded by whitespace/newline)

Reply:
That's good to know. I'll leave it like this then as we need the full stop